native/cocos/platform/SDLHelper.cpp

```diff
@@ -335,44 +353,58 @@ void SDLHelper::pollEvent(bool *quit) {
     }
 }
 
-bool SDLHelper::createWindow(const char *title,
+int SDLHelper::pollEvent(SDL_Event *event) {
+    int cnt = SDL_PollEvent(event);
+    return cnt;
```

Comment:
Could just be one line `return SDL_PollEvent(event);`

Reply:
This function will be removed in future.
See `SystemWindowManager::processEvent` instead.

Reply:
Can remove it now? And event it will be removed in future, can also polish the codes.